src/Microsoft.Tye.Core/ConfigModel/ConfigApplication.cs

```diff
@@ -21,6 +21,8 @@ public class ConfigApplication
 
         public string? Registry { get; set; }
 
+        public string? Network { get; set; }
```

Comment:
@rynowak how do you feel about this? I don't love it being so top level and being specific to containers but I don't have an alternative yet.